components/coding/studentPortal/ProfileDetailCard.tsx

```diff
@@ -40,7 +44,7 @@ function ProfileDetailCard({
         <BadgesDisplayedComponent
           earnedBadges={badges}
```

Comment:
if a user hasn't earned any badges, the UI shows: 
<img width="154" alt="Screen Shot 2023-01-30 at 10 49 22 AM" src="https://user-images.githubusercontent.com/98791180/215525216-3fcb3000-736d-4e76-bcb5-38b178e873a2.png">

need to add some logic to show 0 in this case